lib/addressfinder/bulk.rb

```diff
@@ -25,14 +25,14 @@ def initialize(http:, verification_version:, default_country:)
       end
 
       def cleanse(args={})
-        AddressFinder::Verification.new(args.merge(http: http)).perform.result
+        AddressFinder::Verification.new(**args.merge(http: http)).perform.result
```

Comment:
Do you know if that's ruby 2.7 and below compatible?

Reply:
I've tested with ruby 2.7.5 as well, rspecs and syntax was ok